config/routes.rb

```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
+
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show, :new, :create] do
       resources :comments, only: [:index, :new, :create]
     end
   end
+
```

Comment:
- [ ] KIndly define a default route(root) where your application should start running. Hint :point_down: 

```rb
root "users#index"
```